sdf/bezier.go

```diff
@@ -292,12 +302,12 @@ func (b *Bezier) closure() error {
 		return nil
 	}
 	if len(b.vlist) == 0 || len(b.vlist) == 1 {
-		return errors.New("bad number of vertices")
+		return fmt.Errorf("bad number of vertices")
```

Comment:
It's a minor optimisation. If you don't have any formatting then you don't need to call fmt.Errorf(). errors.New() get's the job done faster.

Reply:
We had a debate amongst ourselves about fmt.Errorf() vs errors.New() -- we have a team member who prefers fmt.Errorf(), so I was sorta caught in between.  I expected you'd prefer errors.New(), but in the case of bezier.go I let fmt.Errorf() prevail because there are a few other spots where you are doing formatting, so fmt is already imported there.

Reply:
reverted these in 4b78bf8